### ISSUE TITLE
Send message to multiple devices

### ISSUE THREAD

**Issue by KoblStone** [firebase/firebase-admin-python#196]
I want to send notifications to specific users using tokens. firebase-admin can't send multiple tokens so I'm using:

```
for obj in queryset:
            token = obj.app_token
            message = messaging.Message(android=android_config, data=data, token=token)
            try:
                messaging.send(message=message, dry_run=False, app=default_app)
            except Exception as ex:
                print('Error %s' % ex)
```

But it seems that "FCM" is blocked due to the fact that I send a lot of requests per second. Because after that I can not get a notification for a while.
Can this be solved?

**Comment by hiranya911** [comment]
I believe there's some on going work to support multicast messaging in the new FCM REST API. This will be supported once that is implemented (although I don't know what the exact timeline is).

**Comment by hiranya911** [comment]
In the meantime, can you use topic messaging to implement your use case?

**Comment by KoblStone** [comment]
No. The list is generated dynamically and I will need to create multiple topics

**Comment by chong-shao** [comment]
Update: as mentioned in #315, we are working on a new batch-send API for Admin SDK.

**Comment by chong-shao** [comment]
Further updates will be posted on https://github.com/firebase/firebase-admin-node/issues/315

**Comment by aaugustin** [comment]
I believe this issue was fixed by #283 and can be closed.

**Comment by chong-shao** [comment]
Thanks for the reminder!
